client/app/components/Modal/Modal.scss

```diff
@@ -22,15 +22,18 @@
   &Box {
     @include setFontSize($size-20);
     @include fadeIn(0.5s);
-    padding: $size-20;
+    padding: $size-30 $size-50;
```

Comment:
Let's change this to 35px in all directions so the spacing around the scrollable area looks more evenly distributed.